docs/statistics.md

```diff
@@ -294,14 +450,16 @@ Missing packets with sequence numbers 8 and 9 will be reported lost with the nex
 The next received packet has sequence number 8. Reorder tolerance value is increased to 2.
 The packet with sequence number 9 is reported lost.
 
+### pktRcvAvgBelatedTime
 
-
-## pktRcvAvgBelatedTime
+TODO: What's this?
 
 Accumulated difference between the current time and the time-to-play of a packet 
 that is received late.
 
-## pktRcvBelated
+### pktRcvBelated
+
+TODO: Revise this, which side, measured over the interval
```

Comment:
Receiver. This is the number of packets that had the sequence behind ACK and were sent uselessly.